pkg/cmd/util/clientcmd/factory.go

```diff
@@ -327,17 +327,6 @@ func (f *Factory) PodForResource(resource string, timeout time.Duration) (string
 			return "", err
 		}
 		return pod.Name, nil
-	case extensions.Resource("jobs"):
```

Comment:
@fabianofranz fyi, dangers of baking types into the CLI.  This is going to hurt new `oc` against old server, but it was already broken/hurting